docs/LearnCalico.md

```diff
@@ -0,0 +1,358 @@
+# Calico networking in a containerized environment
+
+This GitHub repository is the entry point for running a Calico network in a
+containerized environment.
+
+The `calico-docker` repository allows users to easily configure endpoints and 
```

Comment:
Yes I agree - too much too fast.  Also need to tighten up wording... "The calico-docker repository does not allow" doesn't really make sense, I think you mean that the repository contains a command line tool that allows ....